docs/setup_eecs281.md

```diff
@@ -115,13 +160,32 @@ EECS281 Advanced Makefile Help
 Edit these three lines in the Makefile.  Your values might be different, check the project spec for required file names.
 ```make
 UNIQNAME = not_awdeorio
+# .. 
+IDENTIFIER  = not_a_valid_identifier
 # ...
 EXECUTABLE = main
 # ...
-PROJECTFILE = main.cpp
 ```
 {: data-title="Makefile" }
 
+If your main file is not one of the following forms, where EXECUTABLE is the name of the executable specified in the Makefile:
```

Comment:
Is it worth this wall of text?  Would it be easier to follow the instructions if we just tell them modify it?

Reply:
I'm indifferent on which way we do this. My only thinking is that the majority of students probably name their main file either: `main.cpp`, `project4.cpp`, or `amongus.cpp` (where amongus is the executable name and the current project is p4). For that reason, I feel like the majority of students likely wouldn't need to change the PROJECTFILE value.

We also could leave a note saying to change it, but also saying that you may not need to change it if it fits in the regex. WDYT?

Reply:
Here's another take: 3f07ceb04e0df47fd798b1a992afbdd9a5a7657d

> **Shortcut:** You can skip updating `PROJECTFILE` if ...